src/components/connection_handler/src/connection_handler_impl.cc

```diff
@@ -1675,13 +1703,14 @@ bool ConnectionHandlerImpl::IsHeartBeatSupported(
     uint8_t session_id) const {
   LOG4CXX_AUTO_TRACE(logger_);
   sync_primitives::AutoReadLock lock(connection_list_lock_);
-  uint32_t connection = static_cast<uint32_t>(connection_handle);
-  ConnectionList::const_iterator it = connection_list_.find(connection);
-  if (connection_list_.end() == it) {
-    LOG4CXX_WARN(logger_, "Connection not found !");
+  uint32_t connection_id = static_cast<uint32_t>(connection_handle);
```

Comment:
@AKalinich-Luxoft `const`?

Reply:
@AByzhynar fixed in https://github.com/smartdevicelink/sdl_core/pull/3065/commits/5676d3e72f8cf438604136ce4c85efb66e408a38